packages/sol-compiler/package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@0x/sol-compiler",
-    "version": "4.1.1",
+    "version": "4.2.0",
```

```suggestion
    "version": "4.1.1",
```
Will be automatically updated by publish job.